index.js

```diff
@@ -10,7 +10,7 @@ module.exports = {
         var sourceCode = context.getSourceCode();
         var fileContent = sourceCode.getText();
 
-        if (fileContent.indexOf(options.toFind) < 0) {
+        if (fileContent.indexOf(options.toFind) !== -1) {
```

Comment:
The idea with this code is that an error is reported if the given string is not found.  It could potentially be changed to `fileContent.indexOf(options.toFind) === -1` instead, although I'm not seeing the case where it would matter...

Reply:
Ok, So actually It's not an issue, I tought the inverse case, (I didn´t read well the porpouse in the readme)
I  was needed to find a rule like "no-contains" or something like that, because I need to smell code lines and comments for prevent usage of reserved words, but fortunately I found another rule for resolve my problem in the comments https://eslint.org/docs/rules/no-warning-comments

Thanks!

Reply:
Cool, no problem.